src/modules/DataStore/BadgeCase.ts

```diff
@@ -12,19 +12,19 @@ export default class BadgeCase implements Feature {
 
     defaults: Record<string, any> = {};
 
-    badgeList: Array<boolean> = getArrayOfObservables(emptyBadgeList);
+    badgeList: Array<Observable<boolean>> = emptyBadgeList.map((v) => ko.observable(v));
```

Comment:
I had the same thought initially, but TS doesn't like it:
```js
badgeList: Array<Observable<boolean>> = emptyBadgeList.map(ko.observable);
// TS2322: Type 'Observable<unknown>[]' is not assignable to type 'Observable<boolean>[]'

badgeList: Array<Observable<boolean>> = emptyBadgeList.map(ko.observable<boolean>);
// TS1005: '(' expected.
```

We also don't need `ko.observable<boolean>(v)` here because TS will prevent me from passing any other type in as the observable value

Reply:
This works, but it looks ugly
```ts
badgeList = emptyBadgeList.map<Observable<boolean>>(ko.observable);
```
So lets just leave it alone